src/components/transport_manager/include/transport_manager/transport_manager_impl.h

```diff
@@ -124,7 +124,12 @@ class TransportManagerImpl
    *
    * @return Code error.
    */
-  int Init(resumption::LastState& last_state) OVERRIDE;
+  int Init(resumption::LastStateWrapperPtr last_state_wrapper) OVERRIDE;
+
+  DEPRECATED
+  int Init(resumption::LastState&) {
+    return 1;
```

Comment:
Include proper function definition

Reply:
@jacobkeeler Done in 4fdae6f97e8445b4aebe5269c4163e028115f508